test-sbt-testing/js/src/test/scala/zio/test/runner/DummyJSSpec.scala

```diff
@@ -0,0 +1,14 @@
+package zio.test.runner
+
+import zio.test.{DefaultRunnableSpec, Predicate, assert, suite, test}
+
+object DummyJSSpec extends DefaultRunnableSpec({
```

Comment:
You shouldn't need both the parentheses here and the curly braces, just one, and by convention, I'd say use `{ }`.

Reply:
Just `{}` doesn't compile:
```
Error:(5, 28) not enough arguments for constructor DefaultRunnableSpec: (_spec: => zio.test.ZSpec[zio.clock.Clock with zio.console.Console with zio.system.System with zio.random.Random,Nothing,String])zio.test.DefaultRunnableSpec.
Unspecified value parameter _spec.
object DummyJSSpec extends DefaultRunnableSpec {
```
I'll change to just `()`